Makefile

```
@@ -136,7 +137,10 @@ infra-check-compliance-checkov: ## Run checkov compliance checks
 infra-check-compliance-tfsec: ## Run tfsec compliance checks
 	tfsec infra
 
-infra-lint: infra-lint-scripts infra-lint-terraform infra-lint-workflows ## Lint infra code
+infra-lint: infra-lint-markdown infra-lint-scripts infra-lint-terraform infra-lint-workflows ## Lint infra code
+
+infra-lint-markdown: ## Lint Markdown docs for broken links
+	BASEURL=`pwd`; find . -name \*.md -print0 | xargs -0 -n1 markdown-link-check --config .github/workflows/markdownlint-config.json
```

I wish it was possible to add comments in Makefiles since I would have loved to have a comment explaining this line for shell noobs like myself.

Optional suggestion: Maybe move this line to `./bin/lint-markdown.sh` where we can add comments and explain what it's doing so if anyone needs to update it they are better equipped
```suggestion
	./bin/lint-markdown.sh
```

---

Great suggestion! It'll also make it easier to add other markdown lints later if we want. In the process of this change, I discovered `BASEURL` isn't as necessary as I thought. Must default to current working dir or something. Anyways, feel free to take another look and let me know if you have anything thoughts.